examples/astro/src/pages/web-components.astro

```diff
@@ -281,9 +282,10 @@ import Layout from '../layouts/Layout.astro';
 <script src="@cmsgov/design-system/web-components/ds-choice"></script>
 <script src="@cmsgov/design-system/web-components/ds-dropdown"></script>
 <script src="@cmsgov/design-system/web-components/ds-filter-chip"></script>
+<script src="@cmsgov/design-system/web-components/ds-modal-dialog"></script>
```

Comment:
Hey! Looks great—I tested it in storybook, and it appears to work there. I noticed, though, that only the ds-modal-dialog script was imported here. To fully test this component in astro, would we need additional markup and event handling to open and close the modal?

Reply:
Thanks for catching @tamara-corbalt 
Should be fixed in latest commit.